.github/workflows/update_docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Update Docs
```

Comment:
I think this workflow doesn't update docs but rather it checks that docs are updated, right?

Reply:
correct, that could be a better name